configs/config_all.yaml

```diff
@@ -478,7 +480,9 @@ process:
       ignore_pattern: null                                    # whether to ignore sub-strings with specific pattern when computing simhash.
   - image_deduplicator:                                     # deduplicator to deduplicate samples at document-level using exact matching of images between documents.
       method: phash                                           # hash method for image. One of [phash, dhash, whash, ahash]
+      consider_text: false                                    # whether to consider text hash together with image hash when applying deduplication.
   - video_deduplicator:                                     # deduplicator to deduplicate samples at document-level using exact matching of videos between documents.
+      consider_text: false                                    # whether to consider text hash together with video hash when applying deduplication.
```

Comment:
duplicated, remove this line

Reply:
This arg is added to both image_deduplicator and video_deduplicator.